dev.yml

```
@@ -20,3 +20,15 @@ packages:
   - homebrew:
       - swiftlint
       - swiftformat
+      - xcbeautify
+
+commands:
+  build_package:
+    desc: Build the Package
+    run: ./scripts/xcode_run build ShopifyCheckoutSheetKit
```

Worth creating a `build_package` file for consistency sake?
```suggestion
    run: ./scripts/build_package
```

---

Oh fair point - looking again the `test_package` script only runs `xcode_run` now as xcode_run holds all of the logic for sims/xcbeautify

I'm inclined to go the other way and remove `test_package` and just run xcode_run directly, 1 less indirection and all unified on xcode_run feels like an improvement?

Will still leave the `build_samples` script as it bundles in the entitlements steps etc